packages/openapi/src/ui/index.tsx

```diff
@@ -68,7 +68,7 @@ export function API({
     <div
       {...props}
       className={cn(
-        'flex flex-col gap-x-6 gap-y-4 max-lg:[--fd-toc-height:46px] max-md:[--fd-toc-height:36px] xl:flex-row xl:items-start',
+        'flex flex-col gap-x-6 gap-y-4 max-xl:[--fd-toc-height:46px] max-md:[--fd-toc-height:36px] xl:flex-row xl:items-start',
```

Comment:
This small update adjusts the `APIInfo` component's position on screens ranging from 1024px to 1280px.

Before:
![obraz](https://github.com/user-attachments/assets/1def5654-0330-4de3-826b-cbdff318ab74)

After:
![obraz](https://github.com/user-attachments/assets/d680f84f-126d-4bd5-ab3e-437ab68ec349)